poker-pals/src/components/Login.vue

```diff
@@ -104,8 +104,17 @@
       });
 
       // Navigate to tables page if authentication was successful
-      this.socket.on("authenticated", userID => {
-        this.$router.push({ name: "Tables", params: {authenticated: true, userID: userID}});
+      this.socket.on("authenticatedUser", userID => {
+        this.$router.push({ name: "Tables", params: {authenticated: true, socket: this.socket, userID: userID}});
```

Comment:
Have you implemented a single page navigation? From what you've added, it looks like you're still appending to an existing url

Reply:
sometimes i mix up which branch im working on and make changes in one but not the other. all occurrences of 'push' are swapped out with 'replace' in the daily bonus branch. ill add the fix to this branch too